src/server/coprocessor/executor/table_scan.rs

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use server::coprocessor::endpoint::{prefix_next, is_point};
+use server::coprocessor::Result;
+use tipb::executor::TableScan;
+use kvproto::coprocessor::KeyRange;
+use storage::{SnapshotStore, Statistics};
+use util::codec::table;
+use util::collections::HashSet;
+use super::{Executor, Row};
+use super::scanner::Scanner;
+
+struct TableScanExec<'a> {
```

Comment:
I prefer to use the full name rather than a prefix/abbreviation.
s/TableScanExec/TableScanExecutor/